### ISSUE TITLE
Is a payload with a promise required? (Error)

### ISSUE THREAD

**Issue by kodayashi** [pburtchaell/redux-promise-middleware#195]
## Version Number

5.0.0

## Code Example

export const formFactor = () => ({
  type: types.DEVICE_FORM_FACTOR
});

## Expected Behavior

As soon as I add in this middleware the above action doesn't fire anymore. With this middleware, must I always return a payload with a promise?

## Actual Behavior

Nothing happens

**Comment by pburtchaell** [comment]
I can't really help you without any other information. Can you provide the code for your store and the middleware stack you're using? To answer your question, no, you do not always need a promise as the payload. Synchronous actions will dispatch normally.

**Comment by kodayashi** [comment]
Shoot, sorry @pburtchaell just saw this. I had a flood of notifications come in and I missed it. I had to hit a deadline so I swapped out thunk and this library for redux-saga. For some reason that worked, probably an issue of my own. Apologies for not posting more code, I was in a learning state where I didn't really know what I didn't know ;)

It's great to know this middleware supports it. I may go back later for another look when I come up for air. Thanks for the response!
